python_ta/typecheck/type_store.py

```diff
@@ -36,7 +37,15 @@ def __init__(self, type_constraints):
                     if gen == 'Generic':
                         self.classes[class_def.name]['__pyta_tvars'] = tvars
             for function_def in class_def.nodes_of_class(astroid.FunctionDef):
+                poly_tvars = []
                 arg_types = []
+                try:
+                    tvars = self.classes[class_def.name]['__pyta_tvars']
+                    for tvar in tvars:
+                        if isinstance(eval(tvar), TypeVar):
```

Comment:
You can assume that `tvar` always evaluates to a `TypeVar`. Do `eval(tvar, globals())` so that the type variables defined at the top of this file are in scope.